lib/elixir_search_extractor/file_upload/file_uploads.ex

```diff
@@ -32,24 +32,46 @@ defmodule ElixirSearchExtractor.FileUpload.FileUploads do
     end
   end
 
+  def mark_keyword_file_as_completed(keyword_file) do
+    changeset =
```

Comment:
I think this is returning `keyword_file` 💡 

Maybe we could use `Repo.update()` instead of `Repo.update!()` because it is returning {:ok, _} and {:error, _}. It should fit perfectly for multi 👍